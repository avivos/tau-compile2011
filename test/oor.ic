int num1 = 3785430009