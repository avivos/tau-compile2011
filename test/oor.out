1: INT
1: ID(num1)
1: ASSIGN
Line 1: Integer out of range: 3785430009
